pinot-query-planner/src/test/resources/queries/GroupByPlans.json

```diff
@@ -91,6 +91,92 @@
           "\n                  LogicalTableScan(table=[[a]])",
           "\n"
         ]
+      },
+      {
+        "description": "SQL hint based group by optimization with select and aggregate column",
```

Comment:
Do we need to add corresponding tests for the same queries you have used here but without the hint ? Or do they already exist ?

Reply:
It already exists. Added similar tests for newly added cases as well.